bun-tests/fake-snippets-api/fixtures/get-test-server.ts

```diff
@@ -89,6 +89,1560 @@ const seedDatabase = (db: DbClient) => {
     created_at: new Date().toISOString(),
     updated_at: new Date().toISOString(),
   })
+  const snippets = [
+    db.addSnippet({
+      name: "testuser/my-test-board",
+      unscoped_name: "my-test-board",
+      owner_name: "testuser",
+      code: `
+  import { A555Timer } from "@tsci/seveibar.a555timer"
+  
+  export default () => (
+    <board width="10mm" height="10mm">
+      <A555Timer name="U1" />
+    </board>
+  )`.trim(),
+      created_at: new Date().toISOString(),
+      updated_at: new Date().toISOString(),
+      snippet_type: "board",
+      description: "A simple board with an A555 Timer component",
+      circuit_json: [
```

Comment:
Can you generate this circuit json using

import { Circuit } from “@tscircuit/core”

and circuit.getCircuitJson()


Its a bit verbose as is

Reply:
is there any function implementation to generate circuit_json directly with the code?

Reply:
@seveibar i've tried including it, but it needs to fetch the api url to get the packages/modules. I've tried passing api url too, but then due to async calls the db setup in `get-test-server.ts` throws error. should i try reverting this file and just add one snippet in each file as of [here](https://github.com/tscircuit/snippets/pull/237/commits/2186d699db40d5d1b0811432c0293ab8ed1dce92), it passes the remaining 5 tests too.

I know it's not an ideal solution but have been figuring out to make the `getCircuitJson` work for the past one day and i don't find any right approach tbh.